metricbeat/module/ceph/monitor_health/_meta/testdata/input.plain

```diff
@@ -0,0 +1 @@
+{"status": "OK", "output": {"detail": [], "timechecks": {"round_status": "finished", "epoch": 3, "round": 0}, "health": {"health_services": [{"mons": [{"last_updated": "2019-03-13 11:21:24.667025", "name": "26c372192772", "avail_percent": 82, "kb_total": 936145620, "kb_avail": 773013080, "health": "HEALTH_OK", "kb_used": 115509168, "store_stats": {"bytes_total": 4195407, "bytes_log": 4128768, "last_updated": "0.000000", "bytes_misc": 65552, "bytes_sst": 1087}}]}]}, "overall_status": "HEALTH_OK", "summary": []}}
```

Comment:
Why is the suffix of this `.plain`? Looks like json to me.

If you rename this to `docs.json` and remove the `suffix` config in `config.yml` and then run `-generate`, this should directly als update the `data.json` file.

Reply:
@sayden I see you went now with `input.json` instead of `docs.json` as recommended above to have the `data.json` generated. Could you elaborate?

Reply:
I thought that you mean `docs.json` like `foo.json` and that name didn't matter :sweat_smile: In any case, `input` or `doc`, it won't generate a `data.json` named file but a `docs.json-expected.json`, right? Now I'm not sure if I'm missing something else

Reply:
yes: https://github.com/elastic/beats/blob/master/metricbeat/mb/testing/data/data_test.go#L179

Reply:
Ah! Now I understand, I misunderstood completely at the beginning. Thanks for the clarification :)